packages/sdk-node/src/Clerk.ts

```diff
@@ -158,11 +158,12 @@ export default class Clerk extends ClerkBackendAPI {
 
     /** Base initialization */
 
+    // TODO: More comprehensive base initialization
```

Comment:
❓ @igneel64 What do you mean by this comment?

Reply:
This is added on purpose _(commitment)_ and will be addressed in a PR coming up in a bit. 

Currently the Base contructor is position based `new Base(attr1, attr2, etc..)` which does not make it comprehensive enough on which attribute is which without intellisense.
Also kind of in the same feeling, it will make `jwtKey` injectable.

Reply:
Makes sense.